update internships banner (#2971)
DFE-Digital/get-into-teaching-app: app/views/content/teaching-internship-providers.md

      <span class="icon icon-warning" aria-hidden="true"></span>
    </div>
    <div>
      <h2 class="heading-m">Apply from January 2023</h2>
      

>       <p>Applications for the Teaching Internships Programme will open in January 2023. More information on how to apply will be available here, once applications open.</p>

Reviewer: ```suggestion
      <p>Applications for the Teaching Internships Programme will open in January 2023. More information on how to apply will be available on this page, once applications open.</p>
```
Just a small suggestion - feel free to ignore!

>       <p>Applications for the Teaching Internships Programme will open in January 2023. More information on how to apply will be available on this page, once applications open.</p>

Reviewer 2: @philspray I'm not sure we should talk about applying to the 'Teaching Internships Programme' - you don't apply to the programme but to individual schools offering internships. I also think 'programme' is very internal language?

Could we say something like 'You will be able to apply for a teaching internship from January 2023'?

>       <p>Sign up for our emails and we will send you helpful advice and insights about getting into teaching. If you are eligible for the Internships Programme, we will also let you know when applications open.</p>

Reviewer 2: ```suggestion
      <p>Sign up for our emails and we will send you helpful advice and insights about getting into teaching. If you are eligible for a teaching internship, we will also let you know when applications open.</p>
```
